integration-tests/contract-api-scenario.spec.ts

```diff
@@ -74,6 +74,18 @@ CONFIGS.forEach(({ lib, rpc, setup, knownBaker, createAddress, protocol }) => {
       done();
     })
 
+    it('Return undefined when BigMap key is not found', async () => {
+      const myContract = await Tezos.contract.at(knownBigMapContract);
+      const contractStorage: any = await myContract.storage();
+      let value;
+      if (rpc === "https://api.tez.ie/rpc/carthagenet") {
```

Comment:
Could this check unnecessary now? I believe we could use the same PKH for both network and expect the `value` to be undefined?